apps/indexer-coordinator/src/project/project.model.ts

```diff
@@ -213,6 +215,10 @@ export class PaygEntity {
   @Column({ default: 5 })
   @Field()
   overflow: number;
+
+  @Column({ default: '' })
+  @Field()
+  token: string;
```

Comment:
I verified that this field change will not take effect without migration provided. Should I add one for this?